app/views/handlers/list_remove_question.py

```diff
@@ -21,6 +21,14 @@ def is_location_valid(self):
         return True
 
     def handle_post(self):
+        answer_block = self._schema.get_add_block_for_list_collector(  # pylint: disable=protected-access
+            self.parent_block["id"]
+        )
+        answer_id = self._schema.get_first_answer_id_for_block(answer_block["id"])
+        self.questionnaire_store_updater._capture_block_dependencies_for_answer(  # pylint: disable=protected-access
+            answer_id
+        )
```

Comment:
This shouldn't be called outside questionnaire store updated. It's private for a reason, we need to be careful before just disabling these alerts, it signifies something is not right. Make this call inside `remove_list_item_and_answers` if any answer were removed, similar to `update_answers.

Reply:
I am not sure we can move to `remove_list_item_and_answers` of questionnaire store updater as we need ListAction class context to invoke all the methods, we need answer id, parent block/list collector block to extract add block id.

Reply:
Disregard the comment above, moved it to `remove_list_item_and_answers` now.